REFERENCE.md

```diff
@@ -260,6 +260,7 @@ interface JobOpts {
 
   lifo: boolean; // if true, adds the job to the right of the queue instead of the left (default false)
   timeout: number; // The number of milliseconds after which the job should be fail with a timeout error [optional]
+  timeoutFallback: string | Error; // Fallback passed to p-timeout when timeout is specified [optional]
```

Comment:
I think it can be a string or a Function according to p-timeout documentation, right?